### PR TITLE
Add color and size

### PR DESCRIPTION
**Pull request by namuetn**
## Related Tickets:
- [#29918](https://edu-redmine.sun-asterisk.vn/issues/29918)

## WHAT this PR does?
- add productdetail

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by namuetn** [comment]
ready

File: app/Http/Controllers/Admin/ProductController.php [hunk 1]
```diff
@@ -43,10 +44,24 @@ public function create()
      */
     public function store(ProductRequest $request)
     {
+        $colors = $request->colors;
+        $quantities = $request->quantities;
+        // dd($quantities);
```
Comment (thaonp104) [inline app/Http/Controllers/Admin/ProductController.php]: k dùng xoá đi nha

File: resources/views/fashi/admin/product/create.blade.php [hunk 2]
```diff
@@ -61,15 +61,35 @@
...
+                            <option value="yellow">{{ trans('text.yellow') }}</option>
+                            <option value="green">{{ trans('text.green') }}</option>
+                        </select>
+
+                        <label for="size" class="col-form-label">{{ trans('text.size') }}:</label>
+                        <select class="form-control col-md-2 d-sm-inline mt-3 mr-4" name="sizes[]" id="size" value="{{ old('parent_id') }}">
+                            <option value="s">S</option>
+                            <option value="m">M</option>
+                            <option value="l">L</option>
+                            <option value="xs">XS</option>
```
Comment (thaonp104) [inline resources/views/fashi/admin/product/create.blade.php:81]: cho vào config

File: resources/views/fashi/admin/product/create.blade.php [hunk 3]
```diff
@@ -61,15 +61,35 @@
...
+                        </select>
+
+                        <label for="quantity" class="col-form-label">{{ trans('text.in_stock') }}:</label>
+                        <input type="text" class="form-control col-md-2 d-sm-inline mt-3 mr-4  @error('quantity') is-invalid @enderror" id="quantity" name="quantities[]">
+                        @error('quantity')
```
Comment (thaonp104) [inline resources/views/fashi/admin/product/create.blade.php:86]:
```suggestion
                        @error ('quantity')
```

File: resources/views/fashi/admin/product/create.blade.php [hunk 4]
```diff
@@ -61,15 +61,35 @@
...
+                        <label for="size" class="col-form-label">{{ trans('text.size') }}:</label>
+                        <select class="form-control col-md-2 d-sm-inline mt-3 mr-4" name="sizes[]" id="size" value="{{ old('parent_id') }}">
+                            <option value="s">S</option>
+                            <option value="m">M</option>
+                            <option value="l">L</option>
+                            <option value="xs">XS</option>
+                        </select>
+
+                        <label for="quantity" class="col-form-label">{{ trans('text.in_stock') }}:</label>
+                        <input type="text" class="form-control col-md-2 d-sm-inline mt-3 mr-4  @error('quantity') is-invalid @enderror" id="quantity" name="quantities[]">
```
Comment (thaonp104) [inline resources/views/fashi/admin/product/create.blade.php:85]:
```suggestion
                        <input type="text" class="form-control col-md-2 d-sm-inline mt-3 mr-4  @error ('quantity') is-invalid @enderror" id="quantity" name="quantities[]">
```

**Pull request merged**